application/Core/MORR/Data/Capture/Video/Desktop/Utility/NativeDesktop.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace MORR.Core.Data.Capture.Video.Desktop.Utility
+{
+    public static class NativeDesktop
```

Comment:
The convention tends to be to name classes containing p/Invoke imports `NativeMethods` - If you wanted to make clear that this explicitly belongs to this particular module, perhaps name it `DesktopNativeMethods`; otherwise I would prefer just sticking with the more familiar `NativeMethods`, as I think the namespace itself should be enough to disambiguate it from the other classes.

Reply:
Ok I renamed them to `PREFIX`-NativeMethods.